.github/workflows/nodejs.yml

```
@@ -9,7 +9,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x, 18.x, 19.x]
```

Let's leave 19 out for now, it's got a short lifetime and [won't become](https://github.com/nodejs/release#release-phases) a LTS version.

```suggestion
        node-version: [14.x, 16.x, 18.x]
```